napari/layers/base/_constants.py

```diff
@@ -25,3 +25,14 @@ class Blending(StringEnum):
     TRANSLUCENT = auto()
     ADDITIVE = auto()
     OPAQUE = auto()
+
+
+class LayerTypes(StringEnum):
+    LABELS = auto()
+    POINTS = auto()
+    VECTORS = auto()
+    SHAPES = auto()
+    SURFACE = auto()
+    IMAGE = auto()
+    PYRAMID = auto()
```

Comment:
I know that we had `Pyramid` and `Volume` as types at one point - and they appeared in tests in that way, but in some ways that was a historical artifact.

I think we should definitely drop `Volume` from this list and I'm inclined to drop them both and just use `Image` in both cases above. For `Pyramid` we should be able to guess correctly that the data provided is a pyramid and do the right thing even if the `is_pyramid` keyword arg isn't passed

Reply:
As mentioned above, I very much don't see the need at all for this enum. As demonstrated by your comment, @sofroniewn, it's just one more place where things can get out of sync.